apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -236,6 +236,66 @@ export function InstructorQuestionSettings({
                   : ''}
               </table>
             </div>
+            <div class="form-group">
+              <label for="grading_method">Grading Method</label>
+              <select
+                class="form-select"
+                id="grading_method"
+                name="grading_method"
+                value="${resLocals.question.title}"
+                ${canEdit ? '' : 'disabled'}
+              >
+                <option
+                  value="Internal"
+                  ${resLocals.question.grading_method === 'Internal' ? 'selected' : ''}
+                >
+                  Internal
+                </option>
+                <option
+                  value="External"
+                  ${resLocals.question.grading_method === 'External' ? 'selected' : ''}
+                >
+                  External
+                </option>
+                <option
+                  value="Manual"
+                  ${resLocals.question.grading_method === 'Manual' ? 'selected' : ''}
+                >
+                  Manual
+                </option>
+              </select>
+              <small class="form-text text-muted">
+                The grading method used for this question.
+              </small>
+            </div>
+            <div class="form-group form-check">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                id="single_variant"
+                name="single_variant"
+                ${canEdit ? '' : 'disabled'}
+                ${resLocals.question.single_variant ? 'checked' : ''}
+              />
+              <label class="form-check-label" for="single_variant">Single Variant</label>
+              <div class="form-text small text-muted">
+                If checked, the question is not randomized and only generates a single variant.
+              </div>
+            </div>
+            <div class="form-group form-check">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                id="show_correct_answer"
+                name="show_correct_answer"
+                ${canEdit ? '' : 'disabled'}
+                ${resLocals.question.show_correct_answer ? 'checked' : ''}
+              />
+              <label class="form-check-label" for="show_correct_answer">Show Correct Answer</label>
+              <div class="form-text small text-muted">
+                If checked, shows the correct answer panel after submission.
```

Comment:
I think this is an important detail to add:

```suggestion
                If enabled, the correct answer panel will be shown after all submission attempts have been exhausted.
```

Reply:
Committed in [8e108ef](https://github.com/PrairieLearn/PrairieLearn/pull/11290/commits/8e108efc7b763ab17e26beca2d6c23a67eec721f).